README.md

```diff
@@ -356,7 +362,7 @@ User is the claims subject
 ```  
 `Operator` - is an interface responsible for DID document updating
 ```typescript 
-  const userOperator = new Operator(userKeys, resolverSettings);
+  const userOperator = new Operator(userKeys.privateKey, resolverSettings, providerUrl);
```

Comment:
I think this documentation might be incorrect. Should it be?
```suggestion
  const signer = new PrivateKeySigner(privateKey, providerUrl)
  const identityOwner = IdentityOwner.fromPrivateKeySigner(signer)
  const userOperator = new Operator(identityOwner, resolverSettings);
```
(maybe I didn't get it quite right, but I think it should be along these lines?)

Reply:
Yes indeed, I forgot updating those lines. Thank you @jrhender